libcxx/test/support/test_range.h

```diff
@@ -82,4 +83,10 @@ using NonBorrowedView = std::ranges::single_view<int>;
 static_assert(std::ranges::view<NonBorrowedView>);
 static_assert(!std::ranges::borrowed_range<NonBorrowedView>);
 
+template <class Range>
```

Comment:
Should we keep around some kind of static assert to ensure that the test definition of `simple_view` doesn't diverge from the libc++ definition with `__simple_view`?

Reply:
I think that is a *great* idea!

Reply:
I will propose something in the follow up. Sorry for the delay!

Reply:
Let me know what you think.